api.planx.uk/modules/send/email/index.ts

```diff
@@ -97,3 +89,85 @@ export const sendToEmail: SendIntegrationController = async (
     );
   }
 };
+
+const getSubmitEmailConfig = async ({
```

Comment:
This leaves a lot to be desired!

I'm glad we're looking at payload generation soon (in relation to CSV / view applications work). I think if we had each submission generate a valid ODP schema once (and save to the DB), we'd then have a much simpler process of mapping known values whenever we need them.